### PR TITLE
Filecoin Specification Localization by IPFSUNION Team

### PR DESCRIPTION
**Pull request by XQYCHJ**
We plan to create a Chinese version for the Filecoin Spec.

### CODE REVIEW

**Comment by maobisheng** [comment]
https://github.com/XQYCHJ/devgrants/blob/multi-language/open-grant-proposals/open-proposal-filecoin-spec-localization.md

File: open-grant-proposals/open-proposal-filecoin-spec-localization.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Our team has been focusing on Filecoin's mining testing, specification studies, problem submission, and code 
+contributions. 
+
+Our developers have been reading block chain English documents for years and paying close attention to the source
+code, specification and implementation of Filecoin research. 
+
+## Team Translation Example
+
+The translation of Section 2.3 for Filecoin Specification:
+[link of jianshu](https://www.jianshu.com/p/544a2c07366d)
```
Comment (eshon) [inline open-grant-proposals/open-proposal-filecoin-spec-localization.md]: Hi @maobisheng

I'm getting a 404 for https://www.jianshu.com/p/544a2c07366d do you have an updated link for this translation?

Comment (maobisheng) [reply open-grant-proposals/open-proposal-filecoin-spec-localization.md]: Yes, please find below links:
https://juejin.im/post/5e12ec1cf265da5d6f68c83d
https://juejin.im/post/5e12edf4f265da5d363e308d
https://juejin.im/post/5e12ee1ce51d4541375f1389
https://juejin.im/post/5e1300c3e51d454126369bb5

**Comment by eshon** [comment]
Greetings,

Thank you for your application to the Filecoin DevGrants program. We appreciate your proposal and useful starter translations!

Generally we're happy to support Chinese translations of the Filecoin Spec. However the Spec is still being revised so this makes more sense to start this later on. We will let you know when is a good time and look forward to getting help from your team on this.

So this grant is provisionally accepted but will likely not start until later.

Also please note:

请注意，合同不允许将收到开发者津贴用作其他不相关产品或服务的促销，并且不能宣传为合作伙伴企业。

Our DevGrant team will be in touch soon when the Spec is ready for translation.

**Pull request closed without merging**